challenge/admin/challenge_suggestion.py

```diff
@@ -1,9 +1,17 @@
 from django.contrib import admin
 
-from challenge.models import ChallengeSuggestion
+from challenge.models import ChallengeSuggestion, ChallengeSuggestionFeedback
 from config.admin import linkify
 
 
+def reset_counters(self, request, queryset):
+    for challenge_suggestion in queryset:
+        challenge_suggestion.feedback_count = ChallengeSuggestionFeedback.objects.filter(
+            suggestion=challenge_suggestion
+        ).count()
+        challenge_suggestion.save()
```

Comment:
```suggestion
        challenge_suggestion. reset_feedback_count()
```
p1. 모델 메서드를 만들었는, 활용을 안하고 있는 듯?